src/Microsoft.AspNet.FileProviders.Combined/CombinedFileProvider.cs

```diff
@@ -0,0 +1,193 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Extensions.Primitives;
+
+namespace Microsoft.AspNet.FileProviders
+{
+    /// <summary>
+    /// Looks up files using embedded resources in the specified assembly.
+    /// This file provider is case sensitive.
+    /// </summary>
+    public class CombinedFileProvider : IFileProvider
+    {
+        private readonly IFileProvider[] _fileProviders;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="CombinedFileProvider" /> class using a list of file provider.
+        /// </summary>
+        /// <param name="fileProviders"></param>
+        public CombinedFileProvider(params IFileProvider[] fileProviders)
```

Comment:
Hi @davidfowl, is it a general purpose to avoid params, or just in this situation ?